docs/asciidoc/http-client.adoc

```diff
@@ -435,6 +435,42 @@ include::{examplesdir}/metrics/custom/Application.java[lines=18..35]
 <1> Enables HTTP client metrics and provides {javadoc}/reactor/netty/http/client/HttpClientMetricsRecorder.html[`HttpClientMetricsRecorder`] implementation.
 ====
 
+== Tracing
+The HTTP client supports built-in integration with https://micrometer.io/docs/tracing[`Micrometer Tracing`].
+
+The following table provides information for the HTTP client spans:
+
+[width="100%",options="header"]
+|=======
+| contextual name | description
+| HTTP <HTTP METHOD> | Information and total time for the request/response.
+See <<observability-spans-http-client-response-span>>
+| hostname resolution | Information and time spent for resolving the address.
+See <<observability-spans-hostname-resolution-span>>
+| connect | Information and time spent for connecting to the remote address.
+See <<observability-spans-connect-span>>
+| tls handshake | Information and time spent for TLS handshake.
+See <<observability-spans-tls-handshake-span>>
+|=======
+
+The following example enables that integration. This concrete example uses `Brave` and reports the information to `Zipkin`.
+See the https://micrometer.io/docs/tracing#_micrometer_tracing_opentelemetry_setup[`Micrometer Tracing`] documentation for `OpenTelemetry` setup if needed.
+
+====
+[source,java,indent=0]
+.{examplesdir}/tracing/Application.java
+----
+include::{examplesdir}/tracing/Application.java[lines=18..90]
+----
+<1> Initialises Brave, Zipkin and the Observation registry.
```

Comment:
Change `initialises` to `initializes`. (Spring uses American spelling.)

Add a comma after `Zipkin`. (Spring uses serial commas - often called the Harvard comma or the Oxford comma.)